Sources/RxKingfisher/Kingfisher+Rx.swift

```diff
@@ -10,29 +10,29 @@ import RxCocoa
 import RxSwift
 import Kingfisher
 
-extension Reactive where Base == KingfisherWrapper<ImageView> {
+extension Reactive where Base == KingfisherWrapper<KFCrossPlatformImageView> {
```

Comment:
if you have code changes to make you should do these in a separate PR, not this one - just to make sure we're reviewing things seperately. Thanks!